ci-operator/templates/openshift/installer/cluster-launch-installer-kubevirt-e2e.yaml

```diff
@@ -0,0 +1,607 @@
+      - name: PULL_SECRET_PATH
+        value: /etc/openshift-installer/pull-secret
+      - name: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+        value: ${RELEASE_IMAGE_LATEST}
+      - name: OPENSHIFT_INSTALL_INVOKER
+        value: openshift-internal-ci/${JOB_NAME}/${BUILD_ID}
+      - name: USER
+        value: test
+      - name: HOME
+        value: /tmp
+      - name: INSTALL_INITIAL_RELEASE
+        value: ${INSTALL_INITIAL_RELEASE}
+      - name: RELEASE_IMAGE_INITIAL
+        value: ${RELEASE_IMAGE_INITIAL}
+      - name: JOB_NAME
+        value: ${JOB_NAME}
+      command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        # TODO: Wait untill lease is acquired
+
+        if [[ -n "${INSTALL_INITIAL_RELEASE}" && -n "${RELEASE_IMAGE_INITIAL}" ]]; then
+          echo "Installing from initial release ${RELEASE_IMAGE_INITIAL}"
+          OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${RELEASE_IMAGE_INITIAL}"
+        else
+          echo "Installing from release ${RELEASE_IMAGE_LATEST}"
+        fi
+        # poll to make sure that the test container has dropped oc into the shared volume
+        while [ ! command -V oc ]; do sleep 1; done
+        cp "$(command -v openshift-install)" /tmp
+        mkdir ${ARTIFACT_DIR}/installer
+        # TODO: when adding a lease change it
+        #source /tmp/kubevirt-lease
+        source /etc/openshift-installer/kubeconfig-infra-cluster
+        export PATH=$PATH:/tmp
+        export INFRA_CLUSTER_KUBECONFIG=/etc/openshift-installer/kubeconfig-infra-cluster
+        export EXPIRATION_DATE=$(date -d '4 hours' --iso=minutes --utc)
+        export SSH_PUB_KEY=$(cat "${SSH_PUB_KEY_PATH}")
+        export PULL_SECRET=$(cat "${PULL_SECRET_PATH}")
+
+        # TODO: download the kubeconfig
+
+
+        cat > ${ARTIFACT_DIR}/installer/install-config.yaml << EOF
+        apiVersion: v1
+        baseDomain: ${BASE_DOMAIN}
+        metadata:
+          name: ${OCP_CLUSTER}
+        compute:
+        - hyperthreading: Enabled
+          architecture: amd64
+          name: worker
+          platform:
+            kubevirt:
+              cpu: ${WORKER_CPU}
+              memory: ${WORKER_MEM}
+              storageSize: ${WORKER_STORAGE_SIZE}
+          replicas: 3
+        controlPlane:
+          hyperthreading: Enabled
+          architecture: amd64
+          name: master
+          platform:
+            kubevirt:
+              cpu: ${MASTER_CPU}
+              memory: ${MASTER_MEM}
+              storageSize: ${MASTER_STORAGE_SIZE}
+          replicas: 3
+        networking:
+          clusterNetwork:
+          - cidr: ${CLUSTER_NETWORK_CIDER}
+            hostPrefix: ${CLUSTER_NETWORK_HOSTPREFIX}
+          machineNetwork:
+          - cidr: ${MACHINE_NETWORK_CIDER}
+          networkType: ${NETWORK_TYPE}
+          serviceNetwork:
+          - ${SERVICE_NETWORK}
+
+        platform:
+          kubevirt:
+            api_vip: ${KUBEVIRT_APIVIP}
```

Comment:
notice @nargaman is going to rename `api_vip` soon.

Reply:
We talked about it - Nir will change it to be as what I will write